tools/oss-fuzz.sh

```diff
@@ -32,7 +32,13 @@ meson_args=("-Db_lundef=false")
 if [ -z "$FUZZING_ENGINE" ]; then
     meson_args+=("-Dllvm-fuzz=true")
 else
-    meson_args+=("-Doss-fuzz=true" "--auto-features=disabled" "-Dnspawn=enabled" "-Dresolve=true")
+    meson_args+=(
+        "-Doss-fuzz=true"
+        "--auto-features=disabled"
+        "-Dremote=auto"
```

Comment:
To judge from https://github.com/systemd/systemd/actions/runs/15112013722/job/42473443779?pr=37510 it isn't built there. I'm not sure what's going on there.

Reply:
Ah right I forgot about meson's stupid behavior where you can't explicitly set stuff to `auto` if you set `--auto-features=disabled`.